core/services/ocr3/plugins/ccip/commit/factory.go

```diff
@@ -48,22 +51,24 @@ func NewPluginFactory() *PluginFactory {
 
 func (p PluginFactory) NewReportingPlugin(config ocr3types.ReportingPluginConfig,
 ) (ocr3types.ReportingPlugin[[]byte], ocr3types.ReportingPluginInfo, error) {
+	var oracleIDToP2pID map[commontypes.OracleID]libocrtypes.PeerID // TODO: Get this from ocr config, it's the mapping of the oracleId index in the DON
 	onChainTokenPricesReader := reader.NewOnchainTokenPricesReader(
 		reader.TokenPriceConfig{ // TODO: Inject config
 			StaticPrices: map[ocr2types.Account]big.Int{},
 		},
 		nil, // TODO: Inject this
 	)
-
 	return NewPlugin(
 		context.Background(),
 		config.OracleID,
+		oracleIDToP2pID,
 		cciptypes.CommitPluginConfig{},
 		nil,
 		onChainTokenPricesReader,
 		nil,
 		nil,
 		nil,
+		nil,
```

Comment:
Can we add comments to these `nil` fields so we know what they are (until they get fully specified):

```suggestion
		nil, /* blah item 1 */
		onChainTokenPricesReader,
		nil, /* blah item 2 */
		nil,
		nil,
		nil,
```